src/components/productGallery/ProductGallery.css

```diff
@@ -0,0 +1,87 @@
+.product-gallery-main {
+  height: 350px;
```

Comment:
I noticed Airbnb's gallery height is around 447px, so setting it to 447 or 450px should make it look closer to theirs.

Reply:
The main airbnb website is responsive, meaning that on different screens the height and width may change. For example, in the below screenshot, it is less than 400px. However, I change it to 450px.

<img width="599" alt="image" src="https://github.com/user-attachments/assets/af5a97d7-e654-4a60-b2d0-cff01ea7b74c">